tests/wpt/webgl/meta/conformance/textures/svg_image/__dir__.ini

```diff
@@ -0,0 +1 @@
+disabled: SVG support is not implemented yet
```

Comment:
iirc we don't have plans to implement this right now, so if that's the case I don't see the need to file an issue for it